Bridge/Bridge.sol

```diff
@@ -222,41 +317,55 @@ contract Bridge {
         bytes32 txHash,
         uint8[] memory v,
         bytes32[] memory r,
-        bytes32[] memory s)
-    public shouldBeInitialized {
-        require(used[txHash] == false);
-        require(checkSignatures(keccak256(abi.encodePacked(
-                name,
-                symbol,
-                decimals,
-                sidechainAssetId,
-                txHash,
-                _networkId
-            )),
-            v,
-            r,
-            s), "Peer signatures are invalid"
+        bytes32[] memory s
+    ) external shouldBeInitialized {
+        require(used[txHash] == false, "txHash already used");
+        require(
+            checkSignatures(
+                keccak256(
+                    abi.encodePacked(
+                        address(this),
```

Comment:
Also this part (and same code in another functions) is same as in https://github.com/soramitsu/sora2-substrate/pull/644